setup.py

```diff
@@ -140,10 +139,9 @@ def read_requirements(path):
 EXTRAS = {
 
     # Admin
-    'docs': DOCS_REQUIRE,
-    'dev': DEV_REQUIRES + DOCS_REQUIRE + URSULA_REQUIRES + ALICE_REQUIRES,
+    'dev': DEV_REQUIRES + URSULA_REQUIRES + ALICE_REQUIRES,
```

Comment:
Can we keep the 'docs' directive, and just not include it as part of the 'dev' and 'deploy' directives?

Reply:
Unfortunately not, since the dependencies defined in the pip extra are included as part of the distribution's metadata causing twine to fail as shown in #2524

Reply:
I too am am big fan of pip extras, I think the longer term solution here is to either fork `jsonschema2rst`, wait for the maintainer to merge https://github.com/inspirehep/jsonschema2rst/pull/8, or create another custom solution entirely.

Reply:
Ok - perhaps we can revisit once `jsonschema2rst` is updated.